app/backend/src/main/java/com/wishlist/controllers/AuthController.java

```diff
@@ -21,10 +26,14 @@ public class AuthController {
     Logger logger = Logger.getLogger(AuthController.class.getName());
 
     private final IAuth authService;
+    private final IJWTGenerator jwtGenerator;
+    private final IUserService userService;
 
     @Autowired
-    public AuthController(IAuth authService) {
+    public AuthController(IAuth authService, IJWTGenerator ijwtGenerator, IUserService userService) {
```

Comment:
```suggestion
    public AuthController(IAuth authService, IJWTGenerator jwtGenerator, IUserService userService) {
```

Reply:
In Java, interfaces and classes are named in 
interface: InterfaceName
class: InterfaceNameImpl
Convention.
IInterface is used for Typescript mainly. Nothing wrong, just a convention